airbyte-integrations/connectors/source-mailgun/source_mailgun/manifest.yaml

```diff
@@ -39,7 +39,7 @@ definitions:
     type: "DefaultPaginator"
     pagination_strategy:
       type: "CursorPagination"
-      cursor_value: "{{ last_records['paging', 'next'] }}"
```

Comment:
do we know if this even worked? I'd expect `last_records` to be an array, not an object